Detector/DetFCChhHCalTile/compact/FCChh_HCalBarrel_TileCal.xml

```diff
@@ -2,30 +2,34 @@
 <lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
   xmlns:xs="http://www.w3.org/2001/XMLSchema"
   xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
-
   <info name="FCChh_HCalBarrel_TileCal"
-    title="Barrel Calorimeters"
-    author="C. Solans, J. Lingemann"
+    title="Barrel Calorimeter"
+    author="C. Neubueser"
```

Comment:
Maybe keep C. Solans as he actually did studies and his implementation was the basis to kick this off?